pkg/service/replay/absmatch.go

```diff
@@ -0,0 +1,454 @@
+package replay
+
+import (
+	"encoding/json"
+	"strings"
+
+	"go.keploy.io/server/v2/pkg"
+	"go.keploy.io/server/v2/pkg/models"
+	"go.uber.org/zap"
+)
+
+// AbsMatch (Absolute Match) compares two test cases and returns a boolean value indicating whether they are equal or not.
+// It also returns a AbsResult object which contains the results of the comparison.
+// Parameters: tcs1, tcs2, noiseConfig, ignoreOrdering, logger
+// Returns: bool, *models.AbsResult
+func AbsMatch(tcs1, tcs2 *models.TestCase, noiseConfig map[string]map[string][]string, ignoreOrdering bool, logger *zap.Logger) (bool, bool, bool, *models.AbsResult) {
+	if tcs1 == nil || tcs2 == nil {
+		logger.Error("test case is nil", zap.Any("tcs1", tcs1), zap.Any("tcs2", tcs2))
+		return false, false, false, nil
+	}
+
+	pass := true
+	absResult := &models.AbsResult{}
+
+	kindResult := models.StringResult{
```

Comment:
By, default the final result is true, but in case of any mismatch, it will return false. I've handled some other cases like different no of test cases or something in the KTestPilot which I will add asap.